core/types/rollup_cost.go

```diff
@@ -53,13 +53,23 @@ type L1CostFunc func(rcd RollupCostData, blockTime uint64) *big.Int
 // receipts.
 type l1CostFunc func(rcd RollupCostData) (fee, gasUsed *big.Int)
 
+const (
+	// offsets of the packed fields within the L1FeeScalarsSlot
+	BasefeeSlotOffset     = 0
+	BlobBasefeeSlotOffset = 4
+)
+
 var (
 	L1BasefeeSlot = common.BigToHash(big.NewInt(1))
 	OverheadSlot  = common.BigToHash(big.NewInt(5))
 	ScalarSlot    = common.BigToHash(big.NewInt(6))
-)
 
-var L1BlockAddr = common.HexToAddress("0x4200000000000000000000000000000000000015")
+	// slots added for the Ecotone upgrade
+	L1BlobBasefeeSlot = common.BigToHash(big.NewInt(7))
+	L1FeeScalarsSlot  = common.BigToHash(big.NewInt(8)) // basefeeScalar & blobBasefeeScalar are packed in this slot
```

Comment:
This is confusing with the "ScalarSlot" variable. Maybe also mention Ecotone here, rather than only in the `L1BlobBasefeeSlot` doc comment, for rendered docs to be easier to follow?

Reply:
clarified.